controllers/tasks.js

```diff
@@ -367,11 +383,12 @@ const updateTaskStatus = async (req, res, next) => {
     ]);
     taskLog.id = taskLogResult.id;
 
-    if (dev) {
-      if (req.body.percentCompleted === 100) {
-        return next();
-      }
-    }
+    // assignTask function in middleware is not in used and not working
+    // if (userStatusFlag) {
+    //   if (req.body.percentCompleted === 100) {
+    //     return next();
+    //   }
+    // }
```

Comment:
Why comments

Reply:
The 'assignTask' function is hardcoded and has not been utilized anywhere in the codebase. This is because the 'userStatusFlag' is currently being used from the frontend. However, When replacing 'dev' with userStatusFlag results in the 'assignTask' function not working correctly.

<img width="735" alt="image" src="https://github.com/Real-Dev-Squad/website-backend/assets/107163260/59de2e3f-546c-4beb-bafe-1a47dc28dadf">